src/ophys_etl/modules/segmentation/modules/roi_merging.py

```diff
@@ -0,0 +1,88 @@
+import argschema
+import pathlib
+import h5py
+import json
+
+from ophys_etl.modules.segmentation.modules.schemas import \
+    RoiMergerSchema
+
+import ophys_etl.modules.segmentation.postprocess_utils.roi_merging as merging
+from ophys_etl.modules.segmentation.graph_utils.plotting import graph_to_img
+import networkx
+
+import logging
+import time
+
+
+logger = logging.getLogger(__name__)
+logging.captureWarnings(True)
+logging.basicConfig(level=logging.INFO)
+
+
+def write_out_rois(roi_list, out_name):
+    output_list = []
+    for roi in roi_list:
+        new_roi = merging.ophys_roi_to_extract_roi(roi)
+        output_list.append(new_roi)
+
+    with open(out_name, 'w') as out_file:
+        out_file.write(json.dumps(output_list, indent=2))
+
+
+class RoiMergerEngine(argschema.ArgSchemaParser):
+
+    default_schema = RoiMergerSchema
+
+    def run(self):
+
+        diagnostic_dir = None
+        if self.args['diagnostic_dir'] is not None:
```

Comment:
can be handled in the schema by changing from `str` to `OutputDir`
https://argschema.readthedocs.io/en/latest/api/argschema.fields.html#argschema.fields.files.OutputDir

Reply:
Obsolete. This module no longer takes a diagnostic_dir